packages/stack/src/views/Stack/CardContainer.tsx

```diff
@@ -128,6 +130,38 @@ function CardContainer({
 
   const { colors } = useTheme();
 
+  const [pointerEvents, setPointerEvents] = React.useState<'box-none' | 'none'>(
+    'box-none'
+  );
+
+  const valueListenerCallback = React.useCallback(
+    ({ value }: { value: number }) => {
+      const newIsUnderEpsilon = value <= EPSILON ? 'box-none' : 'none';
+      if (newIsUnderEpsilon !== pointerEvents) {
+        setPointerEvents(newIsUnderEpsilon);
+      }
+    },
+    [pointerEvents]
+  );
+
+  const listenerRef = React.useRef<string>();
```

Comment:
You don't need a ref. Move it to the callback to `useEffect`, then you won't need to set it to undefined either.